src/components/NavBar.vue

```diff
@@ -77,7 +77,7 @@
           <div class="navbar-iconWrapper tools-mobile-icon" />
           <span class="nav-mobile-button-text">Tools</span>
         </button>
-        <button class="nav-mobile-button" data-cyId="navbar-openProfile" @click="openProfile">
+        <button class="nav-mobile-button" data-cyId="navbar-editProfile" @click="editProfile">
```

Comment:
Sorry, I was a little confused last time I reviewed this but I think I get the jist now. I think we still want `@click="openProfile"` here since we presumably want mobile to also use the new profile page, but calling `editProfile` means it will open the onboarding modal.

When we do that though, it means the `editProfile` method in this file will never be called, the `emit` from this file will never be used, and the emit will not be received in the parent (`Dashboard`). Could all of that be deleted?